themes/grass/static/plugins/tracking/matomo.js

```diff
@@ -0,0 +1,10 @@
+var _paq = window._paq = window._paq || [];
+_paq.push(['trackPageView']);
```

Comment:
```suggestion
/* Call disableCookies before calling trackPageView */
_paq.push(['disableCookies']);
_paq.push(['trackPageView']);
```

Using `disableCookies`, see https://matomo.org/faq/general/faq_157/